README.md

```diff
@@ -1,10 +1,12 @@
 # EncompassRest
-Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
+Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+ and 2.0.
```

Comment:
We should only display the minimums, thus the plus signs. The fact that there's now a 2.0 specific build doesn't need to be expressed here, they'll see it on NuGet.